### PR TITLE
CXP-1105: Clean webhook part of Connectivity infrastructure

### PR DESCRIPTION
**Pull request by willy-ahva**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: src/Akeneo/Connectivity/Connection/back/tests/EndToEnd/Webhook/ConsumeProductEventEndToEnd.php [hunk 1]
```diff
@@ -120,7 +120,7 @@ public function test_it_sends_a_product_created_webhook_event()
             ]
         );
 
-        /** @var $businessEventHandler BusinessEventHandler */
+        /** @var $businessEventHandler \Akeneo\Connectivity\Connection\Infrastructure\Webhook\MessageHandler\BusinessEventHandler */
```
Comment (guillaumefer-akeneo) [inline src/Akeneo/Connectivity/Connection/back/tests/EndToEnd/Webhook/ConsumeProductEventEndToEnd.php:123]:
```suggestion
        /** @var $businessEventHandler BusinessEventHandler */
```
the same below

**Review by guillaumefer-akeneo — APPROVED** [review]

File: src/Akeneo/Connectivity/Connection/back/tests/EndToEnd/Webhook/ConsumeProductModelEventEndToEnd.php [hunk 2]
```diff
@@ -57,7 +57,7 @@ public function test_it_sends_a_product_model_created_webhook_event()
             ]
         );
 
-        /** @var $businessEventHandler BusinessEventHandler */
+        /** @var $businessEventHandler \Akeneo\Connectivity\Connection\Infrastructure\Webhook\MessageHandler\BusinessEventHandler */
```
Comment (masacc) [inline src/Akeneo/Connectivity/Connection/back/tests/EndToEnd/Webhook/ConsumeProductModelEventEndToEnd.php:60]:
```suggestion
        /** @var BusinessEventHandler $businessEventHandler */
```
same below

File: src/Akeneo/Connectivity/Connection/back/tests/EndToEnd/Webhook/ConsumeProductEventEndToEnd.php (hunk 1 above)
Comment (masacc) [reply src/Akeneo/Connectivity/Connection/back/tests/EndToEnd/Webhook/ConsumeProductEventEndToEnd.php:123]:
```suggestion
        /** @var BusinessEventHandler $businessEventHandler */
```

File: src/Akeneo/Connectivity/Connection/back/tests/EndToEnd/Webhook/IncrementEventsApiRequestCountEndToEnd.php [hunk 3]
```diff
@@ -81,7 +81,7 @@ public function test_it_increments_events_api_request_count()
             ]
         );
 
-        /** @var $businessEventHandler BusinessEventHandler */
+        /** @var $businessEventHandler \Akeneo\Connectivity\Connection\Infrastructure\Webhook\MessageHandler\BusinessEventHandler */
```
Comment (masacc) [inline src/Akeneo/Connectivity/Connection/back/tests/EndToEnd/Webhook/IncrementEventsApiRequestCountEndToEnd.php:84]:
```suggestion
        /** @var BusinessEventHandler $businessEventHandler */
```

**Review by masacc — APPROVED** [review]

**Pull request merged**